core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/DocGeneratorUtil.java

```diff
@@ -254,7 +254,7 @@ static String joinAcceptedValues(List<String> acceptedValues) {
             return "";
         }
 
-        return acceptedValues.stream().collect(Collectors.joining("`, `", "`", "`"));
+        return acceptedValues.stream().map(String::toLowerCase).collect(Collectors.joining("`, `", "`", "`"));
```

Comment:
This isn't the right fix. We need to hyphenate the values.

Are you sure that just removing `@DefaultConverter` didn't fix your issue altogether? From the code, I think it would and you don't need to change anything else.

Reply:
It fixes, thanks. Not sure though why it did not work when I first tried to remove the default converter ...